src/main/java/ir/sahab/kafkaconsumer/SmartCommitKafkaConsumer.java

```diff
@@ -269,6 +276,18 @@ public void start() throws InterruptedException, IOException {
         thread.start();
     }
 
+    /**
+     * Registers metrics and starts JMX reporter.
+     */
+    private void initMetrics() {
+        metricRegistry.register("unappliedAcks", (Gauge) unappliedAcks::size);
+        metricRegistry.register("queuedRecordsSize", (Gauge) queuedRecords::size);
```

Comment:
1. I think fullness metrics (a floating point between 0-1 or a percent number between 0-100) is more readable.

UnappliedAcksFullness
QueuedRecordsFullness

Reply:
Done